docs/start_guide.md

```diff
@@ -0,0 +1,255 @@
+```
+
+### ~/helloapache/Nulecule
+
+This is the `Nulecule` file for Atomic App. The `Nulecule` file is composed of graph and metadata in order to link one or more containers for your application.
+```yaml
+---
+specversion: 0.0.2
+id: helloapache-app
+
+metadata:
+  name: Hello Apache App
+  appversion: 0.0.1
+  description: Atomic app for deploying a really basic Apache HTTP server
+
+graph:
+  - name: helloapache-app
+
+    params:
+      - name: image
+        description: The webserver image
+        default: centos/httpd
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 80
+
+    artifacts:
+      docker:
+        - file://artifacts/docker/hello-apache-pod_run
+      kubernetes:
+        - file://artifacts/kubernetes/hello-apache-pod.json
+      marathon:
+        - file://artifacts/marathon/helloapache.json
+```
+
+##### Spec and id information 
+```yaml
+---
+specversion: 0.0.2
+id: helloapache-app
+```
+
+Here we indicate the specversion of our Atomic App (similar to a `v1` or `v2` of an API) as well as our ID.
+
+##### Metadata
+```yaml
+metadata:
+  name: Hello Apache App
+  appversion: 0.0.1
+  description: Atomic app for deploying a really basic Apache HTTP server
+```
+
+An __optional__ requirement, but this is the metadata associated with your packaged container. A good description as well as a version number will help indicate other users what your Nuleculized container is all about.
+
+
+##### Graph
+
+```yaml
+graph:
+  - name: helloapache-app
+
+    params:
+      - name: image
+        description: The webserver image
+        default: centos/httpd
+      - name: hostport
+        description: The host TCP port as the external endpoint
+        default: 80
+
+    artifacts:
+      docker:
+        - file://artifacts/docker/hello-apache-pod_run
+      kubernetes:
+        - file://artifacts/kubernetes/hello-apache-pod.json
+      marathon:
+        - file://artifacts/marathon/helloapache.json
+```
+
+__Graph__ is the most important section. In here we will indicate all the default parameters as well as all associated artifacts.
+
+```yaml
+- name: image
+  description: The webserver image
+  default: centos/httpd
+```
+There will likely be many parameters that need to be exposed at deployment. It's best to provide defaults whenever possible. Variable templating is used within artifact files. For example: `$image` within `artifacts/kubernetes/hello-apache-pod.json` becomes `centos/httpd`.
+
+Not providing a default variable will require Atomic App to ask the user. Alternatively, an `answers.conf` file can be provided.
+
+```yaml
+artifacts:
+  docker:
+    - file://artifacts/docker/hello-apache-pod_run
+  kubernetes:
+    - file://artifacts/kubernetes/hello-apache-pod.json
+  marathon:
+    - file://artifacts/marathon/helloapache.json
+```
+In order to use a particular provider, a name and file reference(s) are required.
```

Comment:
might want to mention what is in these files.. might be obvious to us, but to the reader, maybe not so much.